### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by xEuathlus**
Add Hodl.finance domain

### CODE REVIEW

**Review by gokaiorg — CHANGES_REQUESTED** [review]
I can't find your new space. Please make sure that you have filled in all required fields and added a strategy. https://snapshot.org/#/hodl.finance

**Comment by xEuathlus** [comment]
resolved it is hft.eth

File: spaces/domains.json [hunk 1]
```diff
@@ -877,6 +877,7 @@
   "dao.bglnode.online": "bgldao.eth",
   "vote.primapes.gg": "primapes.eth",
   "governance.encode.org": "encodeorg.eth",
+  "vote.hodl.finance": "hodl.finance",
```
Comment (gokaiorg) [inline spaces/domains.json:880]: You will need to edit your ENS here
```suggestion
  "vote.hodl.finance": "hft.eth",
```

**Review by gokaiorg — CHANGES_REQUESTED** [review]

**Comment by gokaiorg** [comment]
Closing this PR for now, feel free to open it again once you've done the changes.

**Pull request closed without merging**